invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -52,16 +45,17 @@ def get_name_identifiers(self, obj):
         for identifier in identifiers:
             scheme = identifier["scheme"]
             value = identifier["identifier"]
-            uri = self.URIS.get(scheme)
 
             name_id = {
                 "nameIdentifier": value,
                 "nameIdentifierScheme": scheme.upper(),
```

Comment:
This has to be a mapping as well that defines the correct name identifier scheme value for DataCite.